resource_mock/__init__.py

```diff
@@ -70,20 +65,20 @@ def _previous(items, pivot):
         return result
 
     def _excel_reader(self) -> NoReturn:
+
         sites = [site for site in Site]
-        workbook = load_workbook(filename=os.path.join(self.path, '2018B-2019A Telescope Schedules.xlsx'))
+        workbook = load_workbook(filename=f'{self.path}/2018B-2019A Telescope Schedules.xlsx')
         for site in sites:
             sheet = workbook[site.name]
-            for row in sheet.iter_rows(min_row=2):
+            for row in sheet.iter_rows(min_row=2):                
                 date = row[0].value
-                self.instruments[site][date] = [c.value for c in row[3:]]
+                f2_filter = lambda x: 'Flamingos2' if x == 'F2' else x
```

Comment:
This isn't really a filter so much as a transformation. I would remove it and just do:
```
self.instruments[site][date] = ['Flamingos2' if c.value == 'F2' else c.value for c in row[3:]]
```